webrtc/RTCPeerConnection-helper.js

```diff
@@ -593,8 +593,13 @@ const trackFactories = {
     setInterval(() => {
       ctx.fillStyle = `rgb(${count%255}, ${count*count%255}, ${count%255})`;
       count += 1;
-
       ctx.fillRect(0, 0, width, height);
+      // If signal is set, add a constant-color box to the video frame.
+      if (signal !== null) {
+        ctx.fillStyle = `rgb(${signal}, ${signal}, ${signal})`;
+        ctx.fillRect(10, 10, 20, 20);
+        let pixel = ctx.getImageData(15, 15, 1, 1);
```

Comment:
What is the purpose of `let pixel = ctx.getImageData(15, 15, 1, 1)`?

Reply:
This does look like there's no good reason for it. I'll send a PR to delete this line once this PR is landed.